genapi.py

```diff
@@ -877,6 +880,7 @@ def typesystem_pxd(ts, ns):
 DBTYPES_JS_URL = 'http://fuelcycle.org/arche/dbtypes.js'
 
 def parse_args(argv):
+    """Parses typesystem arguments for code generation."""
```

Comment:
Do the arguments here have to be written about in a parameters section in the docstring?